internal/eval/eval.go

```
@@ -142,3 +142,71 @@ func evalSLEEP(args []string, store *dstore.Store) []byte {
 	time.Sleep(time.Duration(durationSec) * time.Second)
 	return RespOK
 }
+
+func evalPING(args []string, store *dstore.Store) []byte {
```

golangci-lint (1.64.8)

func `evalPING` is unused

(unused)